spec/tapioca/cli/gem_spec.rb

```diff
@@ -1604,30 +1604,23 @@ class User; end
           assert_success_status(result)
         end
 
-        it "advises of removed file(s) and returns exit_status 1" do
+        it "is aware of exclude option and does not error due to removed files" do
```

Comment:
I think the previous test has value. Can we add a new example instead of modifying the existing one?

Reply:
Thanks for the clarification!